single_tech_samples/databricks/sample4_ci_cd/notebook-sparksql/devops/azure-pipelines.yml

```diff
@@ -0,0 +1,46 @@
+# Nutter pipeline for SparkSQL Tests
+
+trigger:
+  branches:
+    include:
+    - single-tech/*
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.5'
+
+- task: configuredatabricks@0
+  displayName: 'Configure Databricks CLI'
+  inputs:
+    url: $(databricks_host)
+    token: $(databricks_token)
+
+- task: deploynotebooks@0
```

Comment:
I'm not sure if this azdo extension is still supported... last commit was 2 years ago.. :|